github/actions_runner_groups.go

```diff
@@ -12,14 +12,16 @@ import (
 
 // RunnerGroup represents a self-hosted runner group configured in an organization.
 type RunnerGroup struct {
-	ID                       *int64  `json:"id,omitempty"`
-	Name                     *string `json:"name,omitempty"`
-	Visibility               *string `json:"visibility,omitempty"`
-	Default                  *bool   `json:"default,omitempty"`
-	SelectedRepositoriesURL  *string `json:"selected_repositories_url,omitempty"`
-	RunnersURL               *string `json:"runners_url,omitempty"`
-	Inherited                *bool   `json:"inherited,omitempty"`
-	AllowsPublicRepositories *bool   `json:"allows_public_repositories,omitempty"`
+	ID                       *int64   `json:"id,omitempty"`
+	Name                     *string  `json:"name,omitempty"`
+	Visibility               *string  `json:"visibility,omitempty"`
+	Default                  *bool    `json:"default,omitempty"`
+	SelectedRepositoriesURL  *string  `json:"selected_repositories_url,omitempty"`
+	RunnersURL               *string  `json:"runners_url,omitempty"`
+	Inherited                *bool    `json:"inherited,omitempty"`
+	AllowsPublicRepositories *bool    `json:"allows_public_repositories,omitempty"`
+	RestrictedToWorkflows    *bool    `json:"restricted_to_workflows,omitempty"`
+	SelectedWorkflows        []string `json:"selected_workflows,omitempty"`
```

Comment:
While you are here, can you please also add the following field that is currently missing according to the official docs?

```go
WorkflowRestrictionsReadOnly *bool `json:"workflow_restrictions_read_only,omitempty"`
```

Reply:
I added this, but I can't actually find documentation on exactly what this param means so I'm unsure which tests cases I should add for it